src/objects/task1.js

```diff
@@ -0,0 +1,9 @@
+let human = {
+    firstName: "Jonas ",
+    lastName: "Jonaitis ",
+    age: " 30 ",
+    isDeveloper: false
+}
+
+let printHuman = (human) => console.log(human['firstName'] + human['lastName'] + (human['age']) + 'is not a developer')
```

Comment:
Try to print using literals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals